http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/DefaultServiceAccountProvider.scala

```diff
@@ -1,36 +1,20 @@
 package org.broadinstitute.dsde.workbench.leonardo.auth
 
+import cats.effect.IO
+import cats.mtl.ApplicativeAsk
 import com.typesafe.config.Config
 import org.broadinstitute.dsde.workbench.leonardo.model.ServiceAccountProvider
-import org.broadinstitute.dsde.workbench.model.{UserInfo, WorkbenchEmail}
 import org.broadinstitute.dsde.workbench.model.google.GoogleProject
-
-import scala.concurrent.{ExecutionContext, Future}
+import org.broadinstitute.dsde.workbench.model.{TraceId, UserInfo, WorkbenchEmail}
 
 /**
   * A ServiceAccountProvider which uses all defaults.
+  * TOOD: is this still needed? Or move this to test code?
```

Comment:
We definitely don't use it in production.. not sure about test code.

Reply:
yea,  I saw it it's referenced in https://github.com/DataBiosphere/leonardo/tree/develop/automation/gce-deploy...